### ISSUE TITLE
Library sends pong twice

### ISSUE THREAD

**Issue by Scorpio93** [TakahikoKawasaki/nv-websocket-client#199]
I noticed an interesting thing that the library periodically tried to send pong twice, also any one of xxxError functions has not been called. My log:
`2020-06-11 23:50:36.976 17910-18013 I/SocketManager: onPingFrame... ----> OPEN
2020-06-11 23:50:36.977 17910-18014 I/SocketManager: onSendingFrame...--> OPEN
2020-06-11 23:50:36.977 17910-18014 I/SocketManager: onFrameSent... --> OPEN
2020-06-11 23:50:40.747 17910-18014 I/SocketManager: onSendingFrame...--> OPEN
2020-06-11 23:50:40.747 17910-18014 I/SocketManager: onFrameSent... --> OPEN`

**Comment by ivanov199311** [comment]
No, it is not: **onSendingFrame** send before and **onFrameSent** aftres frame send
